doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -410,7 +410,10 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`hogp_readme` library:
+
+  * The :c:func:`bt_hogp_rep_read` function forwards the GATT read error code through the registered user callback.
```

Comment:
Is this a new or updated function? Either way, let's use the convention that we use in the changelog like this:

* Updated the ... function to forward...

Reply:
Changed